docs/core/whats-new/index.md

```diff
@@ -0,0 +1,160 @@
+
+In previous versions of the .NET Core, you had to run the [dotnet restore](../tools/dotnet-restore.md) command to download dependencies immediately after you created a new project with the [dotnet new](../tools/dotnet-new.md) command, as well as whenever you added a new dependency to your project. In .NET Core 2.0, `dotnet restore` runs implicitly when the `dotnet new` command executes. It also runs implicitly if dependencies need to be updated when other commands, such as the `run`, `build`, and `publish` commands, execute.
+
+You can also disable the automatic invocation of `dotnet restore` by passing the `--no-restore` switch to the `new`, `run`, `build`, `publish`, `pack`, and `test` commands. 
+
+### Retargeting to .NET Core 2.0
+
+If the .NET Core 2.0 SDK is installed, projects that target .NET Core 1.x can be retargeted to .NET Core 2.0. 
+
+To retarget to .NET Core 2.0, edit your project file by changing the value of the `<TargetFramework>` element (or the `TargetFrameworks>` element if you have more than one target in your project file) from 1.x to 2.0:
+
+```xml
+<PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+ </PropertyGroup>
+```
+
+You can also retarget .NET Standard libraries to .NET Standard 2.0 the same way:
+
+```xml
+<PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+ </PropertyGroup>
+```
+
+For more information about migrating your project to .NET Core 2.0, see [Migrating from ASP.NET Core 1.x to ASP.NET Core 2.0](aspnet/core/migration/1x-to-2x/index).
+
+## Language support
+
+In addition to supporting C# and F#, .NET Core 2.0 also supports Visual Basic.
+
+**Visual Basic**
+
+With version 2.0, .NET Core now supports Visual Basic 2017.  You can use Visual Basic to create the following project types:
+
+- .NET Core console apps
+- .NET Core class libraries
+- .NET Standard class libraries
+- .NET Core unit test projects
+- .NET Core xUnit test projects 
+
+For example, to create a Visual Basic "Hello World" application, do the following steps from the command line:
+
+1. Open a console window, create a directory for your project, and make it the current directory.
+
+1. Enter the command `dotnet new console -lang vb`.
+
+   The command creates a project file with a `.vbproj` file extension, along with a Visual Basic source code file named *Program.vb*. This file contains the source code to write the string "Hello World!" to the console window.
+  
+1.  Enter the command `dotnet run`. The [.NET Core CLI tools](../tools/index.md) automatically compile and execute the application, which displays the message "Hello World!" in the console window.
+
+**Support for C# 7.1**
+
+.NET Core 2.0 supports C# 7.1, which adds a number of new features, including:
+
+- The `Main` method, the application entry point, can be marked with the [async](../../csharp/language-reference/keywords/async.md) keyword.
+- Inferred tuple names.
+- Default expressions.
+- Pattern matching with generics. 
```

Comment:
After talking with the team, we should probably drop this one. See issue #2799 for details and links. (It's more of a bug fix than a new feature, but the spec was also fixed.)